2020-06-17 21:33:11,602 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2020-06-17 21:33:11,603 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Python38\lib\site-packages\instabot\bot\bot.py
2020-06-17 21:33:11,603 - instabot version: 0.117.0 (api_login) - DEBUG - Loading cookies
2020-06-17 21:33:11,604 - instabot version: 0.117.0 (api_login) - DEBUG - Loading uuids
2020-06-17 21:33:11,604 - instabot version: 0.117.0 (api_login) - INFO - Recovery from D:\python_files\InstaBot/config/viivek_uvv_uuid_and_cookie.json: COOKIE True - UUIDs True - TIMING, DEVICE and ...
- user-agent=Instagram 117.0.0.28.123 Android (28/9.0; 420dpi; 1080x1920; OnePlus; ONEPLUS A3003; OnePlus3; qcom; en_US; 180322800)
- phone_id=798076f4-f73d-4a32-a16f-c9c77aba0e74
- uuid=92240e82-69b6-43a5-b3b1-6cb53e26b937
- client_session_id=4c34e53c-64ea-4124-9239-2a704b3dead1
- device_id=android-0134f5818f0fb323
2020-06-17 21:33:11,613 - instabot version: 0.117.0 (api_login) - INFO - LOGIN FLOW! Just logged-in: False
2020-06-17 21:33:12,953 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: feed/timeline/ returned response: <Response [200]>
2020-06-17 21:33:14,108 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: feed/reels_tray/ returned response: <Response [200]>
2020-06-17 21:33:14,117 - instabot version: 0.117.0 (api) - INFO - Logged-in successfully as 'viivek_uvv'!
2020-06-17 21:33:14,118 - instabot version: 0.117.0 (bot_checkpoint) - DEBUG - Loading Checkpoint file from: D:\python_files\InstaBot/config/viivek_uvv.checkpoint
2020-06-17 21:33:14,757 - instabot version: 0.117.0 (api) - DEBUG - GET to endpoint: users/viswa5427/usernameinfo/ returned response: <Response [200]>
2020-06-17 21:33:30,050 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: direct_v2/threads/broadcast/upload_photo/ returned response: <Response [400]>
2020-06-17 21:33:30,050 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message": "challenge_required", "challenge": {"url": "https://i.instagram.com/challenge/?challenge_node_id=18038947249259419", "api_path": "/challenge/?challenge_node_id=18038947249259419", "hide_webview_header": true, "lock": true, "logout": false, "native_flow": true}, "status": "fail"}'
2020-06-17 21:33:30,051 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2020-06-17 21:33:30,051 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2020-06-17 21:33:30,052 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: {}
2020-06-17 21:33:30,053 - instabot version: 0.117.0 (bot_direct) - INFO - Message to ['10140409752'] wasn't sent
2020-06-17 21:33:30,054 - instabot version: 0.117.0 (bot_checkpoint) - DEBUG - Saving Checkpoint file to: D:\python_files\InstaBot/config/viivek_uvv.checkpoint
2020-06-17 21:33:30,056 - instabot version: 0.117.0 (bot) - INFO - Total messages: 1/76
2020-06-17 21:33:30,061 - instabot version: 0.117.0 (bot) - INFO - Total requests: 62
